app/javascript/chat/draw/index.jsx

```diff
@@ -1,18 +1,24 @@
 import { h } from 'preact';
 import { useRef, useEffect, useState } from 'preact/hooks';
 import PropTypes from 'prop-types';
+import { chatDrawPalettes } from '../../utilities/Constants';
```

Comment:
non-blocking. It's fine to import them, but all I meant was they could be in the same file, just outside of the `<Draw />` component.